WordPress/src/main/java/org/wordpress/android/models/Note.java

```diff
@@ -390,13 +390,51 @@ private JSONObject getCommentActions() {
         return mActions;
     }
 
-    /*
+    @NonNull
+    private JSONObject getPostActions() {
```

Comment:
This is copied from getCommentActions(). It looks really bad, I plan to use Gson to refactor this part when we move Note to fluxc project.